s4e-backend/src/test/java/pl/cyfronet/s4e/controller/InvitationControllerTest.java

```diff
@@ -173,6 +241,17 @@ public void shouldReject() throws Exception {
         assertEquals(invitationAfterReject.get().getStatus(), InvitationStatus.REJECTED);
     }
 
+    @Test
+    public void deleteShouldBeSecured() throws Exception {
```

Comment:
Please verify the happy path of delete as well.

Reply:
```
    @Test
    public void shouldSendEmailOnDeletionAndRemoveInvitation() throws NotFoundException {
        val invitation = InvitationHelper
                .invitationBuilder(InvitationHelper.institutionBuilder().build())
                .build();
        when(invitationService.findByToken(invitation.getToken(), Invitation.class))
                .thenReturn(Optional.of(invitation));

        listener.handle(new OnDeleteInvitationEvent(invitation.getToken(), null));

        verify(mailService).sendEmail(eq(invitation.getEmail()), any(), any(), any());
        verify(invitationService).deleteBy(eq(invitation.getToken()));
    }

Reply:
This doesn't touch DB and verify authorization.